index.php

```diff
@@ -145,13 +169,15 @@ function showGrades($ruser, $csvFile)
     $htmlout .= "There is a problem with the handback dir... ";
 }
 ?>
-<!doctype html>
+<!DOCTYPE html>
 <html>
-<head>
+  <head>
     <meta charset="utf-8">
     <title>Handback</title>
-
-</head>
+    <style>
+    <?php include 'table.css'; ?>
```

Comment:
This should be a `link` or inline like @rxg suggested.  I think for small amounts of CSS it's overkill to have a separate file. If it starts getting large and complicated it's good to keep them separate.

Reply:
My thought is to actually have a 'default' css file that users modify. Now I'm not sure people will want to, but...
We don't want the average user making changes to code that is under version control.

Reply:
So my vote is for link